scripts/TCL/positionData/434246_2023-02-08.json

```diff
@@ -0,0 +1,8 @@
+{
+    "amount0": 19.107256692287137,
+    "amount1": 81715.67414189917,
+    "liquidity": 10045740743300893339638,
+    "lowerTick": 59700,
+    "tokenId": 434246,
```

Comment:
we don't own [434246](https://app.uniswap.org/#/pool/434246), iirc this data was generated from a tx that never actually went through. our token id for gtc/eth id is [435187](https://app.uniswap.org/#/pool/435187)